### PR TITLE
FEATURE: CL-8484 Use lodash for utility methods where possible

### PR DESCRIPTION
**Pull request by akloboucnik**

### CODE REVIEW

File: test/project_test.js [hunk 1]
```diff
@@ -63,7 +64,7 @@ describe('project', () => {
                     [200, {'Content-Type': 'application/json'},
                     JSON.stringify({query: {entries: [{}, {}]}})]
                 );
-                project.getDatasets('myFakeProjectId').then(function(result) {
+                project.getDatasets('myFakeProjectId').then(function resolve(result) {
```
Comment (voy) [inline test/project_test.js]: yay! named functions ftw

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/18/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/19/

**Comment by voy** [comment]
Fixup and merge?

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/20/

**Pull request merged**